src/scss/cdb-components/buttons.scss

```diff
@@ -0,0 +1,165 @@
+// Buttons styles
+// ----------------------------------------------
+
+/* SG
+# Buttons/Primary
+
```

Comment:
Could we add a description explaining which state classes we use (for example) or other relevant information related with these new buttons?